content/en/docs/troubleshooting.md

```diff
@@ -3,4 +3,26 @@ title: "Troubleshooting"
 description: "Troubleshooting common problems with Notary"
 type: docs
 weight: 7
----
\ No newline at end of file
+---
+
+## How do i troubleshoot issues with signatures?
+
+Inspect the descriptor of a remote artifact sent for signing using  `oras manifest fetch`. For example:
+
+```console
+$ oras manifest fetch --descriptor IMAGE:TAG
+{"mediaType":"...","digest":"sha256:...","size":...}
+```
+
+Confirm the details, such as the *mediaType* and *digest* match the details of the artifact sent for signing.
+
+Inspect the signatures of a remote artifact using `notation inspect` with the digest value for the artifact. For example:
+
+```console
+$ notation inspect IMAGE@sha256:111222333444555666777888999000aaaabbbbcccddddeeeeffff00011122233
+Inspecting all signatures for signed artifact
+IMAGE@sha256:111222333444555666777888999000aaaabbbbcccddddeeeeffff00011122233
+└── application/vnd.cncf.notary.signature
+```
+
+Confirm the expected signatures are present.
```

Comment:
Maybe we can add more details here for inspecting signatures .

For example, the output of `notation inspect` command contains
- certificate information, users can validate the signing certificates and certificate chain.
- signed artifact, which should contain the same descriptor information as the output from `oras manifest fetch`, to confirm the correct artifact is being signed.

Reply:
thank you for the feedback @yizha1 

Updated the section to incorporate the additional details